app/styles/site-views/search-list.scss

```diff
@@ -0,0 +1,80 @@
+$app-qgreen: rgba(98,144,128,0.25);
+$app-halfgreen: rgba(98,144,128,0.5);
+$app-lightgray: rgb(239,239,239);
+
+.centerdiv-list-wrapper {
+  background-color: $app-lightgray;
+}
+
+.centereddiv-list {
+
```

Comment:
centereddiv-list should be a different name. classes always have names in this format: first-second-third (with - ) between. also no need to say center-div just call it center-list

Reply:
changed names